node/cli/src/cli.rs

```diff
@@ -30,6 +31,17 @@ pub struct RunCmd {
     pub force_vara: bool,
 }
 
+impl RunCmd {
+    /// Set `Wasm` as default execution strategy.
+    pub fn with_wasm_execution(&mut self) {
+        self.base
+            .import_params
+            .execution_strategies
+            .execution
+            .get_or_insert(ExecutionStrategy::Wasm);
```

Comment:
get_or_insert won't work if Native already set, isn't it?

Reply:
Cannot understand the reason of creating this method